src/Zir.zig

```diff
@@ -62,8 +60,8 @@ pub const ExtraIndex = enum(u32) {
 };
 
 pub fn getMainStruct(zir: Zir) Inst.Index {
-    return zir.extra[@enumToInt(ExtraIndex.main_struct)] -
-        @intCast(u32, Inst.Ref.typed_value_map.len);
+    _ = zir;
+    return 0;
 }
```

Comment:
hmm I suggest this:

`pub const main_struct_index = 0;`

this is simpler than a function call with an unnecessary argument but if things change to work a different way in the future we will still get a compile error at the usage sites.